Tests/PlayingCard/SuitTests.swift

```diff
@@ -0,0 +1,35 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+*/
+
+@testable import PlayingCard
+import XCTest
+
+class SuitTests: XCTestCase {
+    
+    func testSuitStringEquality(){
+        let c:Suit = .clubs
+        let d:Suit = .diamonds
+        let h:Suit = .hearts
+        let s:Suit = .spades
```

Comment:
I would write these inline, there isn't a need to put them in separate variables.